src/components/CommentItem.tsx

```diff
@@ -0,0 +1,35 @@
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comment: Comment;
+  deleteComment: (commentId: number) => void;
+};
+
+export const CommentItem: React.FC<Props> = ({ comment, deleteComment }) => {
+  const handleCommentDeleteClick = () => {
+    deleteComment(comment.id);
+  };
+
+  return (
+    <article className="message is-small" data-cy="Comment">
+      <div className="message-header">
+        <a href={`mailto:${comment?.email}`} data-cy="CommentAuthor">
+          {comment?.name}
+        </a>
+        <button
+          data-cy="CommentDelete"
+          type="button"
+          className="delete is-small"
+          aria-label="delete"
+          onClick={handleCommentDeleteClick}
+        >
+          delete button
```

Comment:
The text 'delete button' inside the button element is redundant and may confuse users. Consider removing this text, as the button's purpose is already clear from its class and aria-label.